omnigibson/action_primitives/starter_semantic_action_primitives.py

```diff
@@ -453,12 +561,13 @@ def _convert_cartesian_to_joint_space(self, target_pose):
             )
         return joint_pos, control_idx
     
-    def _target_in_reach_of_robot(self, target_pose):
+    def _target_in_reach_of_robot(self, target_pose, is_relative=False):
+        if not is_relative:
+            target_pose = self._get_pose_in_robot_frame(target_pose)
```

Comment:
Create a separate function (_target_in_reach_of_robot, _target_in_reach_of_robot_relative) for this instead of taking in an argument. Also document both.